docs/concepts/samplers.rst

```diff
@@ -0,0 +1,61 @@
+.. _samplers_sdk:
+
+=======================
+Samplers and Composites 
+=======================
+
```

Comment:
Have you considered putting links at the top (i.e. the line here), so that users can just jump to the different subsections of this page easily?

Reply:
They do have: 
```
.. _samplers_sdk:

=======================
Samplers and Composites 
=======================
```
Do you mean something else?

Reply:
I had meant links to `Samplers`, `Hybrid Quantum-Classical Samplers` etc. But as you had already pointed out, these exist on the side panel. So please ignore my earlier suggestion.